ethbloom/Cargo.toml

```diff
@@ -0,0 +1,25 @@
+[package]
+name = "ethbloom"
+version = "0.6.0"
+authors = ["Parity Technologies <admin@parity.io>"]
+description = "Ethereum bloom filter"
+license = "MIT"
+documentation = "https://docs.rs/ethbloom"
+homepage = "https://github.com/paritytech/primitives"
+repository = "https://github.com/paritytech/primitives"
+
```

Comment:
`homepage`  and `repository` needs to be updated to https://github.com/paritytech/parity-common